artemis-cli/src/main/java/org/apache/activemq/artemis/cli/commands/tools/PrintData.java

```diff
@@ -216,63 +216,63 @@ private static void printPages(DescribeJournal describeJournal,
 
          if (pgStore != null) {
             folder = pgStore.getFolder();
```

Comment:
There's a semantic change here, are you sure?

Reply:
The not-null check here on `pgStore` indicates that `pgStore` may, in fact, be null. If it is null then it will trigger a `NullPointerException` almost straight away.